common/transport/transport-api/src/main/java/org/thingsboard/server/common/transport/util/JsonUtils.java

```diff
@@ -56,7 +60,14 @@ public static JsonElement parse(Object value) {
         } else if (value instanceof Long) {
             return new JsonPrimitive((Long) value);
         } else if (value instanceof String) {
-            return JsonParser.parseString((String) value);
+            try {
```

Comment:
this looks so strange for JsonUtils

Reply:
It cannot parse Base64 properly

Reply:
It cannot parse Base64 properly

Reply:
Can you share an example when it useful?

Reply:
testWriteReplaceValueMultipleResource_Result_CHANGED_Multi_Instance_Resource_must_One_ValueJsonNodeToBase64()
![image](https://github.com/user-attachments/assets/b9b54d7d-afde-49a1-9177-b1eb850215dc)